integrations/services/cms_integration.py

```diff
@@ -219,7 +219,34 @@ def should_add(self):
         return True
 
 
-CMS_INTEGRATIONS = {"brevo": BrevoIntegration, "hubspot": HubSpotIntegration}
+class NcHubSpotIntegration(HubSpotIntegration):
```

Comment:
What would you think about going ahead and making HubSpotIntegration an abstract class, and CO and NC could have their own implementations? Could keep the CMS_INTEGRATIONS dict the same so we don't affect other places.